commands/approve.js

```diff
@@ -1,13 +1,14 @@
+require("dotenv").config();
 const embedService = require("../services/embedService");
 const db = require("../models").sequelize;
 const mediaModel = db.models.Media;
 require("../services/channelBindingService").ChannelBinding;
 
+const approverRole = process.env.hasOwnProperty("APPROVER_ROLE") ? process.env.APPROVER_ROLE : "Moderator";
+
 const getApproval = async (msg, args, options, resource) => {
   const reactionFilter = (reaction, user) => {
-    return (
-      ["✅", "❌"].includes(reaction.emoji.name) && user.id === msg.author.id
-    );
+    return ["✅", "❌"].includes(reaction.emoji.name) && user.id === msg.author.id;
```

Comment:
Maybe it would be worth extracting the reactions to constants for quick config (check/crossmark by default) in case someone would want to use different emoji, like 👎  / 👍 or custom sets

Reply:
If someone really needs to change the reactions used then sure, works.